Tests/AsyncHTTPClientTests/RequestValidationTests.swift

```diff
@@ -0,0 +1,95 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the AsyncHTTPClient open source project
+//
+// Copyright (c) 2018-2019 Apple Inc. and the AsyncHTTPClient project authors
```

Comment:
@fabianfett can we fix the copyright here? should be `2020` because it's all new code

Reply:
fixed